src/commands/Moderation/BanCommand.ts

```diff
@@ -0,0 +1,97 @@
+import BaseCommand from "#base/BaseCommand";
+import { inject, injectable } from "tsyringe";
+import { kClient, kPrisma } from "#utils/tokens";
+import { Client, CommandInteraction, Permissions, MessageEmbed, GuildTextBasedChannel } from "discord.js";
+import { ModLogCase } from "#utils/ModLogCase";
+import { ModLogCaseType } from "#utils/constants";
+import type { PrismaClient } from "@prisma/client";
+
+@injectable()
+export default class extends BaseCommand {
+    constructor(@inject(kClient) public readonly client: Client<true>, @inject(kPrisma) public prisma: PrismaClient) {
+        super({
+            name: "ban",
+            category: "Moderation"
+        });
+    }
+
+    async execute(interaction: CommandInteraction) {
+        await interaction.deferReply({ ephemeral: true });
+
+        if (!interaction.memberPermissions?.has(Permissions.FLAGS.BAN_MEMBERS)) {
+            return await interaction.followUp({ content: "You don't have the required permissions to run this command", ephemeral: true });
+        }
+
+        const server = await this.prisma.guild.findFirst({
+            where: {
+                id: interaction.guildId
+            }
+        });
+
+        if (!server || !server.modlog || !interaction.guild?.channels.cache.has(server.modlog)) {
+            return await interaction.followUp({ content: "No mod log channel found for this server" });
```

Comment:
We don't want the users to compulsorily have mod logs enabled, do we?

The users should be able to ban member without having the modLog enabled.